build/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.10.0
```

Comment:
I've gone back and forth on this - I think at one point Kevin asked me not to put it in the changelog :)